turbopy/diagnostics.py

```diff
@@ -302,6 +317,10 @@ def __init__(self, owner: Simulation, input_data: dict):
         self.field_was_found = False
 
         self.outputter = None
+        self.csv = None
+        self.csv_input = self._input_data.get("filename", None)
```

Comment:
See comments above about using `self.outputter` instead of `self.csv`. You don't need these lines.

Reply:
```suggestion
```